ref/general/about-open-liberty.adoc

```diff
@@ -0,0 +1,80 @@
+Application code in Open Liberty runs in a single thread pool that is called the default executor.
+The size of this pool is set by an auto-tuning controller, which can manage a wide range of workloads.
+
+=== Zero migration deployment
+
+Zero migration is the ability to use existing, unmodified user files with an updated version of the product runtime files, with no unwanted or unexpected change in behavior. Open Liberty does not modify user configuration files, which are fully compatible between versions.
+You can use a single version of your configuration files across multiple versions.
+Files that you created for a previous version of Open Liberty can be used with a later version.
+Files that you create for later versions can be used with previous versions.
+As a result, if all configured features are installed, you can use a single set of configuration files across multiple versions without modifications.
+
+=== Separating configuration from code
+
+Throughout the development, testing, and production lifecycle of your application, you can easily specify changes to configuration according to the deployment context. With Microprofile Config, you can build your app once and deploy it many times through your CI/CD pipeline by changing the configuration for each deployment. For more information, see link:/guides/microprofile-config.html[Configuring microservices].
+
+
+== Monitor applications in the cloud
+
+Distributed tracing helps you track the flow of a request as it moves through an application.
+Tracing creates observability, so you can debug errors and better understand throughput. With Open Liberty, it's simple to implement a standard system to trace requests through an application and record the results to customizable logs.
+
+=== Customizing logs
+You can specify custom fields in your Open Liberty logs to capture detailed information about specific requests, users, or behaviors. Custom fields can help sort and filter your logs, so you can better understand what's happening in your application. For more information, see link:https://openliberty.io/blog/2019/12/03/custom-fields-json-logs.html[
+Adding custom fields to JSON logs in Open Liberty].
+
+=== Gathering and analyzing metrics
+You can link:/docs/ref/general/#microservice_observability_metrics.html[gather detailed metrics on your applications] that run on Open Liberty with MicroProfile metrics. Metrics are essential to building observability into applications so that operations teams can monitor application behavior effectively. After you configure metrics, you can link:/blog/2020/01/29/alerts-slack-prometheus-alertmanager-open-liberty.html[get alerts from your Open Liberty apps in Slack using Prometheus Alertmanager]. You can also link:blog/2020/01/23/Kibana-dashboard-visualizations.html[compile the data from your logs into visualizations in dashboards like Kibana], so you can spot potential problems in your Open Liberty server in real time.
```

Comment:
"You can also link:blog/2020/01/23/Kibana-dashboard-visualizations.html[compile the data from your logs into visualizations in dashboards like Kibana], so you can spot potential problems in your Open Liberty server in real time."
Here, change "so you can spot potential problems in your Open Liberty server in real time." to "to spot potential problems in your Open Liberty server in real time.", to avoid repetition of 'so' in one sentence.